soccer/Robot.cpp

```diff
@@ -587,12 +618,51 @@ void OurRobot::replanIfNeeded(const Geometry2d::CompositeShape& global_obstacles
 	if (!_pathInvalidated) {
 		addText("Reusing path", Qt::white, "Planning");
 	} else {
-		Planning::Path *path = _planner->run(pos, angle, vel, _motionConstraints, &full_obstacles);
-		
+		double leadTime = *(_motionConstraints._replan_lead_time) * 1000000;
```

Comment:
Can't we use a function macro or somethign instead of multiplicative constants? Its really sort of weird.